src/service/vaults.ts

```diff
@@ -75,15 +75,17 @@ const watchPriceFeeds = () => {
 };
 
 type VaultSubscribers = {
-  asset: string;
   vault: string;
 };
 
 /**
  * Ex. `asset: 'published.vaultFactory.manager0'` -> `'manager0'`.
  */
 const getManagerIdFromSubscribers = (subscribers: VaultSubscribers) =>
-  subscribers.asset.split('.').pop();
+  subscribers.vault.split('.').find(node => node.includes('manager'));
```

Comment:
makes sense. NTS to put something like this into inter-protocol/tools so we have a test that ensures it.